scripts/gen_docs.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -e
+
+# Author: Thomas Bechtold <tbechtold@suse.com>
```

Comment:
You may want to adjust/add to this authorship depending on your relative contribution

Reply:
added myself, don't really want to remove an author attribution, though Tom might want me to if he sees how I've mangled his script ;-)